pandas/tests/series/test_repr.py

```diff
@@ -259,6 +259,16 @@ def test_float_repr(self):
         assert repr(ser) == expected
 
 
+    def test_different_null_objects(self):
+        # GH#45263
+        ser = pd.Series([1,2,3,4], [True, None, np.nan, pd.NaT])
+        result = repr(ser)
+        assert "NaT" in result
```

Comment:
Could you assert the entire `result` repr?

Reply:
Done